packages/tools/devtools/devtools-browser-extension/package.json

```diff
@@ -34,6 +34,8 @@
 		"test": "npm run test:mocha",
 		"test:coverage": "nyc npm test -- --reporter xunit --reporter-option output=nyc/junit-report.xml",
 		"test:e2e": "start-server-and-test start:tinylicious 7070 start:client",
+		"test:jest": "jest --detectOpenHandles",
```

Comment:
Based on the docs, seems like we should probably start by not using `--detectOpenHandles`? Or did we need it explicitly?

Reply:
We will need this one to detect async processes: server, browser launch close safely after tests.